src/lib/services/workflow-service.ts

```diff
@@ -214,6 +225,33 @@ export async function cancelWorkflow(
   });
 }
 
+export async function signalWorkflow({
+  namespace,
+  workflowId,
+  runId,
+  signalName,
+  input,
+}: SignalWorkflowOptions) {
+  const route = await routeForApi('workflow.signal', {
+    namespace,
+    workflowId,
+    runId,
+  });
+
```

Comment:
The one thing I think we'll need for this is if they have a codec endpoint set up, we'll need to send the payloads there first to encode before sending to the signal endpoint. 

We've only done decode up to this point, but it should be similar code.